plugins/main/bridge/README.md

```diff
@@ -52,3 +52,8 @@ If the bridge is missing, the plugin will create one on first use and, if gatewa
 * `hairpinMode` (boolean, optional): set hairpin mode for interfaces on the bridge. Defaults to false.
 * `ipam` (dictionary, required): IPAM configuration to be used for this network. For L2-only network, create empty dictionary.
 * `promiscMode` (boolean, optional): set promiscuous mode on the bridge. Defaults to false.
+* `vlan` (int, optional): assign VLAN tag. Defaults to none.
+
+*Note:* The VLAN parameter config the VLAN tag on the host end of the veth and also enable the vlan_filtering feature on the bridge interface.
+
+*Note:* To configure up link for L2 network you need to allow the vlan on the up link interface by using the follow command ``` bridge vlan add vid VLAN_ID dev DEV``` 
```

Comment:
"up link" -> "uplink"
"follow" -> "following"

And I assume DEV here is like eth0 or whatever the hosts physical NIC would usually be?

Reply:
> And I assume DEV here is like eth0 or whatever the hosts physical NIC would usually be?

Correct. A better text would be

To configure up link for L2 network you need to allow the vlan on the up link interface `DEV` by using the follow command

or maybe use the blog example of `bridge vlan add dev bond0 vid 2 master`